go/vm/types.go

```diff
@@ -29,8 +32,42 @@ func (a *Address) UnmarshalText(data []byte) error {
 	return textToBytes(a[:], data)
 }
 
+func (v Value) ToBig() *big.Int {
+	return new(big.Int).SetBytes(v[:])
+}
+
+func (v Value) ToUint256() *uint256.Int {
+	return new(uint256.Int).SetBytes(v[:])
+}
+
 func (v Value) String() string {
-	return fmt.Sprintf("0x%x", v[:])
+	return v.ToUint256().String()
+}
+
+func (v Value) Cmp(o Value) int {
+	return v.ToUint256().Cmp(o.ToUint256())
+}
+
+func ValueFromUint64(value uint64) Value {
+	return ValueFromUint256(new(uint256.Int).SetUint64(value))
```

Comment:
```suggestion
	return ValueFromUint256(value.ToUint256())
```
Would this also work ?

Reply:
No, the `value` here has type `uint64`.